administrator/language/de-DE/plg_system_webauthn.ini

```diff
@@ -4,49 +4,49 @@
-PLG_SYSTEM_WEBAUTHN_FIELD_LABEL="W3C Web-Authentifizierung (WebAuthn) Anmeldung"
-PLG_SYSTEM_WEBAUTHN_FIELD_N_AUTHENTICATORS_REGISTERED="%d Authentifikatoren wurden bereits eingerichtet: %s."
-PLG_SYSTEM_WEBAUTHN_FIELD_N_AUTHENTICATORS_REGISTERED_0="Es wurde noch kein Authentifikator eingerichtet."
-PLG_SYSTEM_WEBAUTHN_FIELD_N_AUTHENTICATORS_REGISTERED_1="Authentifikator wurde bereits eingerichtet: %2$s."
-PLG_SYSTEM_WEBAUTHN_HEADER="W3C Web-Authentifizierung (WebAuthn) Anmeldung"
-PLG_SYSTEM_WEBAUTHN_LBL_DEFAULT_AUTHENTICATOR="Generischer Authentifikator"
+PLG_SYSTEM_WEBAUTHN_FIELD_DESC="Ermöglicht die Verwaltung passwortloser Anmeldeverfahren unter Verwendung von Passkeys. Benötigt wird ein unterstützter Browser und ein Passkey (z.&#8239;B. Google Chrome oder Firefox mit einem FIDO2-zertifizierten Sicherheitsschlüssel).<br><br><strong>MacOS/iOS/watchOS:</strong> Touch/Face ID.<br><strong>Windows:</strong> Hello (Fingerabdruck/Gesichtserkennung/PIN).<br><strong>Android:</strong> Biometrische Displaysperre.<br><br>Weitere Informationen in der <a href='https://docs.joomla.org/Special:MyLanguage/WebAuthn_Passwordless_Login' target='_blank' rel='noopener noreferrer'>Dokumentation zur passwortlosen Passkey-Anmeldung</a>."
+PLG_SYSTEM_WEBAUTHN_FIELD_LABEL="Passkey-Anmeldung"
+PLG_SYSTEM_WEBAUTHN_FIELD_N_AUTHENTICATORS_REGISTERED="%d Passkeys wurden bereits eingerichtet: %s."
+PLG_SYSTEM_WEBAUTHN_FIELD_N_AUTHENTICATORS_REGISTERED_0="Es wurde noch kein Passkey eingerichtet."
+PLG_SYSTEM_WEBAUTHN_FIELD_N_AUTHENTICATORS_REGISTERED_1="Passkey wurde bereits eingerichtet: %2$s."
+PLG_SYSTEM_WEBAUTHN_HEADER="Passkey-Anmeldung"
+PLG_SYSTEM_WEBAUTHN_LBL_DEFAULT_AUTHENTICATOR="Generischer Passkey"
 PLG_SYSTEM_WEBAUTHN_LBL_DEFAULT_AUTHENTICATOR_LABEL="%s wurde am %s hinzugefügt"
-PLG_SYSTEM_WEBAUTHN_LOGIN_DESC="Ohne Passwort mit dem W3C Web-Authentifizierungsstandard (WebAuthn) in kompatiblen Browsern anmelden. Die WebAuthn-Authentifizierung muss vorab im Benutzerprofil eingerichtet werden."
-PLG_SYSTEM_WEBAUTHN_LOGIN_LABEL="Web-Authentifizierung"
-PLG_SYSTEM_WEBAUTHN_MANAGE_BTN_ADD_LABEL="Neuen Authentifikator hinzufügen"
+PLG_SYSTEM_WEBAUTHN_LOGIN_DESC="Ohne Passwort mit dem Passkey in kompatiblen Browsern anmelden. Die Passkey-Authentifizierung muss vorab im Benutzerprofil eingerichtet werden."
+PLG_SYSTEM_WEBAUTHN_LOGIN_LABEL="Passkey"
```

Comment:
Core: `PLG_SYSTEM_WEBAUTHN_LOGIN_LABEL="Sign in with a passkey"`
![image](https://github.com/joomlagerman/joomla/assets/66922325/97cfab70-1d45-4007-8b1f-9691b5a89689)

Reply:
Ja, furchtbar lang.

Wir haben heute auch nur "Web-Authentifizierung"
![Bildschirmfoto 2023-09-12 um 16 35 38](https://github.com/joomlagerman/joomla/assets/1752513/57d1f1bc-e631-46ac-a7d3-335294e3ae0b)

Reply:
Ist ja jetzt etwas kürzer - da wäre evtl. `Passkey verwenden` möglich, damit es nicht "nur" ein englischer Begriff ist.